html/html-forms/html-forms_Finding-a11y-errors/index.html

```diff
@@ -17,15 +17,22 @@ <h1 class="title">Understanding A11y in Forms</h1>
       </p>
 
       <form for="title">
-        <fieldset>
+        <fieldset for="description">
           <legend>Enter your information</legend>
-          <label aria-describedby="name-input">Name:</label>
+          <label for="name-Input" aria-describedby="name-input">Name:</label>
           <input id="name-Input" type="text" name="name" />
```

Comment:
I'm also not sure if the name attribute should or not use the same value as the ID. I mean, it will work as it is, I just wonder if for the accessibility it would be better to have the same value.